proto/ibc/core/channel/v1/tx.proto

```diff
@@ -68,6 +68,7 @@ message MsgChannelOpenInit {
 // MsgChannelOpenInitResponse defines the Msg/ChannelOpenInit response type.
 message MsgChannelOpenInitResponse {
   string channel_id = 1 [(gogoproto.moretags) = "yaml:\"channel_id\""];
+  string version = 2 [(gogoproto.moretags) = "yaml:\"version\""];
```

Comment:
Minor nit: do you mind running `make proto-all` to ensure the proto formatter runs as well as `proto-gen` target :) 

You can also remove the yaml tags on this field, we tend only use add these for fields containing `_` in proto, so for `version` its unnecessary.